sdk/spring/spring-messaging-azure-servicebus/src/main/java/com/azure/spring/messaging/servicebus/core/DefaultServiceBusNamespaceProcessorFactory.java

```diff
@@ -292,7 +293,7 @@ public void addBuilderCustomizer(String entityName,
                                      String subscription,
                                      AzureServiceClientBuilderCustomizer<ServiceBusClientBuilder.ServiceBusProcessorClientBuilder> customizer) {
         if (customizer == null) {
-            LOGGER.debug(LOG_IGNORE_NULL_CUSTOMIZER, ServiceBusClientBuilder.ServiceBusProcessorClientBuilder.class.getName());
+            LOGGER.debug(LOG_IGNORE_NULL_DEDICATED_CUSTOMIZER, ServiceBusClientBuilder.ServiceBusProcessorClientBuilder.class.getName());
```

Comment:
I'd prefer not using any static variable of log format for such case, plain text is much easier to read.